### ISSUE TITLE
create_env.sh doesn't work on hennings machine

### ISSUE THREAD

**Issue by sylvinite** [Clinical-Genomics/MUTANT#76]
**What needs to be done:**
Get the installation working on Hennings machine.

**Suggestions on how to get it done:**
Install on Hennings machine. Fix stuff step by step as they get done.
https://github.com/Clinical-Genomics/MUTANT/blob/main/mutant/standalone/create_env.sh doesnt seem to be working

**What are the arguments for getting it done:**
Make 100% sure that the MUTANT toolbox is developable for everyone.

**Task is considered finished when:**
Henning can install MUTANT on his machine

**Comment by pbiology** [comment]
In general the installation of hasta should be fixed. This is one part of it.

**Comment by henrikstranneheim** [comment]
Yes, I think it was fixed when we upgraded conda. I think it can be closed and reopened if needed
